src/utils/root_key_utils/src/root_key_util.c

```diff
@@ -62,10 +62,9 @@ CryptoKeyHandle MakeCryptoKeyHandleFromADUC_RootKey(const ADUC_RootKey* rootKey)
         key = RSAKey_ObjFromModulusBytesExponentInt(modulus->buffer, modulus->size, rootKey->rsaParameters.e);
         break;
 
-    case ADUC_RootKey_KeyType_INVALID:
-        break;
-
+    case ADUC_RootKey_KeyType_INVALID: // fallthrough
```

Comment:
Fallthroughs can be hard to maintain if someone doesnt pay attention to them when adding new cases.